pkg/api/v1beta2/dynakube/convert_from.go

```diff
@@ -4,16 +4,13 @@ import (
 	"encoding/json"
 
 	"github.com/Dynatrace/dynatrace-operator/pkg/api"
-	v1beta2 "github.com/Dynatrace/dynatrace-operator/pkg/api/v1beta2/dynakube"
+	v1beta3 "github.com/Dynatrace/dynatrace-operator/pkg/api/v1beta3/dynakube"
```

Comment:
(or change it to have the version in the v1beta1 package)
```suggestion
	"github.com/Dynatrace/dynatrace-operator/pkg/api/v1beta3/dynakube"
```